frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 //Import components that you create here
+import NavBar from './Navbar/Navbar'
 
 function LandingPage(){
     return(
         //Use components that you import here
-        <></>
+        <NavBar/>
```

Comment:
Since the NavBar isnt associated with the Landing page can we put this inside of App.js instead?

Reply:
I moved the NavBar to app.js and also moved the whole folder down in components to make it more organized